src/pages/crudPage.tsx

```diff
@@ -1,10 +1,31 @@
 import { NavLink } from "react-router-dom";
 import { Crud } from "../components/Crud";
 
+export type User = {
+  name: string;
+  surname: string;
+  id: string;
+};
+export type UserList<User> = [User, ...User[]];
```

Comment:
Dieses Type brauchen wir hier nicht.  Es soll möglich sein, keine `User`s zu haben, also können wir einfach `User[]` benutzten um die Liste von Users zu beschreiben.

Auch: Ich finde es macht mehr Sinn, wenn das Type `User` in `Crud.tsx` definiert ist, also nicht hier in pages sondern in `components/Crud.tsx`.  Ich glaube ich brauche ein Diagram um das zu erklären, wir können das morgen diskutieren!